pkg/diagnose/connectivity/core_endpoint.go

```diff
@@ -71,7 +71,8 @@ func getLogsUseTCP() bool {
 func Diagnose(diagCfg diagnose.Config, log log.Component) []diagnose.Diagnosis {
 
 	// Create domain resolvers
-	keysPerDomain, err := utils.GetMultipleEndpoints(pkgconfigsetup.Datadog())
+	config := pkgconfigsetup.Datadog()
+	keysPerDomain, err := utils.GetMultipleEndpoints(config)
```

Comment:
Doesn't seem like anything has changed here. Would it make sense to remove this hunk?